helpers/target.go

```diff
@@ -377,107 +378,170 @@ func IsAWSAccReachable(accARN, assumeRoleURL, role string, sessDuration int) (bo
 		assumeRoleResp.SessionToken), nil
 }
 
-// IsDockerImgReachable returns wether the input Docker image exists
-// in registry. Void user and pass does not produce an error and will
-// interact with registry without authentication.
+// IsDockerImgReachable returns whether the input Docker image exists in the
+// registry. Void user and pass does not produce an error unless a token can be
```

Comment:
I'm not sure if I understand this doc line. Is it s/unless/as long as ?

Reply:
You are totally right, it's saying the opposite I wanted to say :D